pkgs/top-level/python-packages.nix

```diff
@@ -1870,6 +1870,24 @@ in modules // {
     };
   };
 
+  beancount-pygments-lexer = buildPythonPackage (rec {
+    version = "0.1.2";
+    name = "beancount-pygments-lexer-${version}";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/b/beancount-pygments-lexer/${name}.tar.gz";
+      sha256 = "0qil39ckf6jhlgjwlxjrfphmx5zs9zhp0b92mrw99kbi06f1qcdq";
+    };
+
+    buildInputs = with self; [ pygments ];
```

Comment:
Nope, I'm not. I would need to change it to `propagatedBuildInputs`, am I right?